containers/jersey-servlet-core/src/main/java/org/glassfish/jersey/servlet/WebComponent.java

```diff
@@ -423,20 +423,20 @@ private void initContainerRequest(
             final HttpServletResponse servletResponse,
             final ResponseWriter responseWriter) throws IOException {
 
-        try {
-            requestContext.setEntityStream(new InputStreamWrapper() {
-                @Override
-                protected InputStream getWrapped() {
-                    try {
-                        return servletRequest.getInputStream();
-                    } catch (IOException e) {
-                        throw new UncheckedIOException(e);
-                    }
+        boolean waitForInputEnable = requestContext.resolveProperty(ServletProperties.WAIT_FOR_INPUT, Boolean.TRUE);
+        long waitForInputTimeOut = requestContext.resolveProperty(ServletProperties.WAIT_FOR_INPUT_TIMEOUT,
+                ServletProperties.WAIT_FOR_INPUT_DEFAULT_TIMEOUT);
+        requestContext.wrapEntityInputStream(new ServletEntityInputStream(waitForInputEnable,
+                waitForInputTimeOut) {
+            @Override
+            protected ServletInputStream getWrappedStream() {
+                try {
+                    return servletRequest.getInputStream();
```

Comment:
Probably clashes with https://github.com/eclipse-ee4j/jersey/pull/5904

Reply:
yes it does but it will be resolved during or after the merge of the 2.x/3.0 into the 3.1